<?xml version="1.0"?>
<launch>
	<node name="reactive_control_node" type="reactive_control_single_integrator" pkg="reactive_control" respawn="false" output="log">
		<rosparam file="$(find reactive_control)/config/reactive_params_single_integrator.yaml"/>
	</node>
</launch>